build-tools/src/main/resources/software/amazon/awssdk/checkstyle.xml

```diff
@@ -387,6 +387,11 @@
 
         <!-- Checks that no 'final' are added on local variables -->
         <module name="software.amazon.awssdk.buildtools.checkstyle.UnnecessaryFinalOnLocalVariableCheck" />
+
+        <!-- Checks that no forbidden methods are used in implementation -->
+        <module name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodCheck">
```

Comment:
Can we use `Regexp` instead?  That way, we don't need to add another dependency

Example: https://github.com/aws/aws-sdk-java-v2/blob/8f361fde4800451d99dce4a1a22c5557fee61aee/build-tools/src/main/resources/software/amazon/awssdk/checkstyle.xml#L361-L365

Reply:
Thanks for the reference. Changes done.